airplane.py

```diff
@@ -32,6 +31,18 @@ def reset():
 
     return 'DB HAS BEEN RESET'
 
+# test create new customer
+@app.route('/customer')
```

Comment:
@katieli16 it would be good to create endpoints that can test the db functions like this and make sure that they are working before going forward